src/wooting-rgb-sdk.c

```diff
@@ -226,7 +226,8 @@ bool wooting_rgb_array_update_keyboard() {
     }
 
     if (rgb_buffer4_changed &&
-        wooting_usb_get_meta()->device_type == DEVICE_KEYBOARD) {
+        (wooting_usb_get_meta()->device_type == DEVICE_KEYBOARD ||
+         wooting_usb_get_meta()->device_type == DEVICE_KEYBOARD_60)) {
```

Comment:
Hmmm, I don't think this is needed as the 60HE uses the v2 interface so never ends up down here. Then even if it did use the v1 interface it prob wouldn't be using the 4th buffer anyway due to the smaller set of keys